src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/string.txt

```diff
@@ -291,4 +291,25 @@ true
 "ab"
 
 >> Concatenate(Text(1/0, "000"), "b", Text(7))
-#Error
\ No newline at end of file
+#Error
+
+>> $"Hello {"World!"}"
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

A few more ... 

$"Hello"
- this should test that new 1 arg overload of Concat.

$"{123}"
- testing cases with empty literals. Plus testing with values that need to get coerced to string (do we support that?). 

$"{123}{456}"
- testing immediately consecutive open and close.

Reply:
also, can we test error conditions, like:

$"{" // missing close
$"}" // missing open

(maybe not in .txt , but somewhere)

Reply:
Added all of these tests.